policyengine_us/variables/gov/states/ms/tax/income/deductions/ms_deductions.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_us.model_api import *
+
+
+class ms_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Mississippi deductions"
```

Comment:
We need a source since some states tie itemization to the federal itemization choice

Reply:
This source indicates that a filer can itemize in MS even if they take the standard federally: 
>In the event you filed using the standard deduction on your federal return and wish to itemize for Mississippi purposes

https://www.dor.ms.gov/sites/default/files/Forms/Individual/80108228.pdf#page=1

But we need to check the reverse as well. A legal code citation would be best.

Reply:
There is a legal code Miss. Code. R. § 3-02-11-103 (https://www.law.cornell.edu/regulations/mississippi/35-Miss-Code-R-SS-3-02-11-103) that covers the reverse scenario.